sesman/sesman.ini.in

```diff
@@ -21,7 +21,14 @@ TerminalServerAdmins=tsadmins
 AlwaysGroupCheck=false
 ; When RestrictOutboundClipboard=true clipboard from the
 ; server is not pushed to the client.
+; In addition, you can control text/file/image transfer restrictions
+; respectively. It also accepts comma separated list such as text,file,image.
 RestrictOutboundClipboard=false
+; When RestrictInboundClipboard=true clipboard from the
+; client is not pushed to the server.
+; In addition, you can control text/file/image transfer restrictions
+; respectively. It also accepts comma separated list such as text,file,image.
+RestrictInboundClipboard=false
```

Comment:
TODO: Don't for get to edit man pages, too.

Reply:
Definitely leave this for now, along with unit tests.

Reply:
Updated man page too.